src/js/components/CheckBox/CheckBox.js

```diff
@@ -201,6 +205,18 @@ const CheckBox = forwardRef(
         checked={checked}
         onClick={stopLabelClick}
         pad={pad}
+        onMouseEnter={event => {
+          if (onMouseEnter) onMouseEnter(event);
```

Comment:
Another option is to use optional chaining - `onMouseEnter={event => onMouseEnter?.(event)}`

Reply:
Great feedback @bojan88 , Thank you!